aws/cloudformation/standalone/gen_ai_curriculum/monitoring/alarms/configurations/openaiSafetyHighFailureRate.ts

```diff
@@ -14,11 +14,11 @@ export const openaiSafetyHighFailureRateConfiguration: PutMetricAlarmInput = {
     OKActions: [],
     AlarmActions: [SNS_TOPIC],
     InsufficientDataActions: [],
-    EvaluationPeriods: 5,
-    DatapointsToAlarm: 5,
+    EvaluationPeriods: 3,
+    DatapointsToAlarm: 2,
     Threshold: 10,
     ComparisonOperator: ComparisonOperator.GreaterThanThreshold,
-    TreatMissingData: 'missing',
+    TreatMissingData: 'ignore',
```

Comment:
Still unclear on the distinction between "missing" and "ignore" but this seems like a good idea regardless 😄

Reply:
haha for sure...the last row in this table is what convinced me to switch. "X" is an above alarm threshold data point, "-" is missing data, and "0" is below alarm threshold. If I'm understanding correctly, the "ignore" approach wouldn't alarm when we have a single above alarm data point and the rest is missing.

<img width="625" alt="image" src="https://github.com/user-attachments/assets/a77b67c7-782f-4137-8a3c-557b8fb278f8" />